### PR TITLE
Issue #405: Fix broken adc get ecmd

### PR DESCRIPTION
**Pull request by AnDann**
Fix for #404 and #405

### CODE REVIEW

**Comment by AnDann** [comment]
I try to fix the bit-macro instead.

**Comment by AnDann** [comment]
As proposed from @eku fix at caller of macro

File: hardware/adc/adc_ecmd.c [hunk 1]
```diff
@@ -62,7 +62,7 @@ parse_cmd_adc_get(char *cmd, char *output, uint16_t len)
     adc = adc_get(channel);
   adc_out:
     output[0] = NIBBLE_TO_HEX(LO4(HI8(adc)));
-    output[1] = NIBBLE_TO_HEX(HI4(adc));
+    output[1] = NIBBLE_TO_HEX(HI4(LO8(adc)));
```
Comment (eku) [inline hardware/adc/adc_ecmd.c]: This solution solves the problem, but it hides the fact, that the LO8 expects an uint8_t as argument.

`output[1] = NIBBLE_TO_HEX(LO8((uint8_t)adc)));`

Comment (AnDann) [inline hardware/adc/adc_ecmd.c]: Thats exactly what LO8 does
```c
#define LO8(x)  ((uint8_t)(x))
```

**Pull request merged**